v2/src/components/appInfoForm/index.tsx

```diff
@@ -33,14 +33,59 @@ type State = {
     netlifyApiRouteUsed: boolean,
     showWebsiteBasePath: boolean,
     showAPIBasePath: boolean
+    /** Whether current element is the first visible AppInfoForm */
```

Comment:
<img width="770" alt="Screenshot 2022-09-05 at 10 22 21 AM" src="https://user-images.githubusercontent.com/18233774/188365695-967c08d4-cae3-43dc-8841-616df3895a58.png">

<img width="911" alt="Screenshot 2022-09-05 at 10 22 28 AM" src="https://user-images.githubusercontent.com/18233774/188365712-f35d1547-ea9f-4132-a1f7-01eede1976cf.png">

The form does not save all the inputs to storage

Reply:
its fixed